### DISCUSSION TITLE
Logical negation on the if/while statement parenthesis

### DISCUSSION THREAD

**Discussion by Saleca** [dotnet/csharplang#6723] [Language Ideas] [answered]
I don know much about the core workings of the language, but I believe it would be very useful to have this feature.

Example:
 I have 2 numbers and a boolean, I wanna do something if the statement is false:
`count < limit && isActive`
- One could:
`if (!(count < limit && isActive))`
- But it seems so natural if we could:
`if !(count < limit && isActive)`

In short I see this as an elegant solution to negate a conditional statement where there is already a mandatory  parenthesis.

**Comment by HaloFour** [comment, marked as answer]
See: #882

**Reply by Saleca** [reply]
how could i miss that. i swear i searched x)
upvoted..
